src/scss/design-system-interface.scss

```diff
@@ -79,6 +77,7 @@ $weight-xbold: 900;
     grid-template-columns: repeat(auto-fit, 15rem);
     grid-gap: 1.3rem;
     justify-content: space-around;
+
     @media (min-width: 960px) {
```

Comment:
I'm not sure what you breaking point variables look like or if this fits into any that's already created because it's being used in multiple spots.